webapp/components/description/style.less

```diff
@@ -1,5 +1,5 @@
-@import (reference) "../app-styles/config";
-@import (reference) "../app-styles/buttons";
+@import (reference) "../../app-styles/config";
+@import (reference) "../../app-styles/buttons";
```

Comment:
Is it possible to use relative imports in less?

Reply:
yes